manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -236,7 +236,7 @@ spec:
       dnsPolicy: "Default"
       containers:
         - name: csi-attacher
-          image: k8s.gcr.io/sig-storage/csi-attacher:v3.4.0
+          image: gcr.io/k8s-staging-sig-storage/csi-attacher:v3.5.0
```

Comment:
Should all be from sig-storage instead of k8s-staging-sig-storage

Reply:
Yes, the images were not available yesterday. Changed it to `sig-storage` as they seem to be available now